docs/Configuration.md

```diff
@@ -390,7 +390,7 @@ An array of directory names to be searched recursively up from the requiring mod
 
 Default: `["js", "json", "jsx", "ts", "tsx", "node"]`
 
-An array of file extensions your modules use. If you require modules without specifying a file extension, these are the extensions Jest will look for.
+An array of file extensions your modules use. If you require modules without specifying a file extension, these are the extensions Jest will look for, in left-to-right order.
```

Comment:
We could have the second paragraph here? something like

>If you are using TypeScript, you may want to consider moving `"ts"` and/or `"tsx"` earlier in the array. Where you place these is up to you - we recommend placing the extensions most commonly used in your project on the left.

?

Reply:
Thanks - I added a second paragraph though I worded it a little differently, let me know if it looks alright to you.

Reply:
perfect!